zproject/settings.py

```diff
@@ -160,7 +160,6 @@ def get_secret(key):
 # of pairs of (setting name, default value that it must be changed from)
 REQUIRED_SETTINGS = [("EXTERNAL_HOST", "zulip.example.com"),
                      ("ZULIP_ADMINISTRATOR", "zulip-admin@example.com"),
-                     ("ADMIN_DOMAIN", "example.com"),
```

Comment:
If we're removing that here, we should also remove it in a few other places:
* `local_settings_template.py`
* Discussion about changing it in scripts/setup/initialize-database

and change the default in `zproject/settings.py` to `zulip.com`.